### PR TITLE
chore: add two new fields to the plugin install generation process

### LINKED ISSUE #1309: Add new operator arguments to operator
Following this PR https://github.com/cloudnative-pg/cloudnative-pg/pull/843 we should add those new options to the generation of the install yaml file of the operator in the plugin

### PR DESCRIPTION
**Pull request by sxd**
We added to new fields to the command line:
* --log-field-level
* --log-field-timestamp

Both used to change the respective values in the operator deployment.

Closes #1309

Signed-off-by: Jonathan Gonzalez V <jonathan.gonzalez@enterprisedb.com>

### CODE REVIEW

File: internal/cmd/plugin/install/generate.go [hunk 1]
```diff
@@ -98,7 +102,7 @@ func newGenerateCmd() *cobra.Command {
 	cmd.Flags().Int32Var(
 		&replicas,
 		"replicas",
-		0,
+		1,
```
Comment (jsilvela) [inline internal/cmd/plugin/install/generate.go:105]: hm, why are we doing this?

Comment (sxd) [reply internal/cmd/plugin/install/generate.go:105]: The default is 1, why not make it the default also in the options?

**Review by jsilvela — APPROVED** [review]

File: internal/cmd/plugin/install/generate.go (hunk 1 above)
Comment (gbartolini) [reply internal/cmd/plugin/install/generate.go:105]: I think that's because we rely on the source manifest, which could come with a different value.

Comment (gbartolini) [inline internal/cmd/plugin/install/generate.go:105]:
```suggestion
		0,
```

I would not change this in this patch - assuming it makes sense to change it.

**Pull request merged**